{
	"schema_version": "1.6.0",
	"id": "CVE-2021-47086",
	"modified": "2024-09-18T03:17:18.977545Z",
	"published": "2024-03-04T18:15:07Z",
	"related": [
		"SUSE-SU-2024:2894-1",
		"SUSE-SU-2024:2939-1",
		"SUSE-SU-2024:2947-1"
	],
	"details": "In the Linux kernel, the following vulnerability has been resolved:\n\nphonet/pep: refuse to enable an unbound pipe\n\nThis ioctl() implicitly assumed that the socket was already bound to\na valid local socket name, i.e. Phonet object. If the socket was not\nbound, two separate problems would occur:\n\n1) We'd send an pipe enablement request with an invalid source object.\n2) Later socket calls could BUG on the socket unexpectedly being\n   connected yet not bound to a valid object.",
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.10.92-1"
						}
					]
				}
			],
			"versions": [
				"5.10.46-4",
				"5.10.46-5",
				"5.10.70-1",
				"5.10.70-1~bpo10+1",
				"5.10.84-1",
				"5.10.92-1~bpo10+1"
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2021-47086.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.15.15-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2021-47086.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.15.15-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2021-47086.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/0bbdd62ce9d44f3a22059b3d20a0df977d9f6d59"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/311601f114859d586d5ef8833d60d3aa23282161"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/48c76fc53582e7f13c1e0b11c916e503256c4d0b"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/52ad5da8e316fa11e3a50b3f089aa63e4089bf52"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/53ccdc73eedaf0e922c45b569b797d2796fbaafa"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/75a2f31520095600f650597c0ac41f48b5ba0068"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/982b6ba1ce626ef87e5c29f26f2401897554f235"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/b10c7d745615a092a50c2e03ce70446d2bec2aca"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2021-47086"
		}
	]
}
